python/dolfinx/wrappers/mesh.cpp

```diff
@@ -301,8 +301,12 @@ void declare_mesh(nb::module_& m, std::string type)
       [](const dolfinx::mesh::Mesh<T>& mesh, int dim,
          nb::ndarray<const std::int32_t, nb::ndim<1>, nb::c_contig> entities)
       {
-        return dolfinx::mesh::create_submesh(
+        auto [submesh, e_map, v_map, g_map] = dolfinx::mesh::create_submesh(
             mesh, dim, std::span(entities.data(), entities.size()));
+        auto _e_map = as_nbarray(e_map);
```

Comment:
I think this will mean arrays are created three times. It could be restructured to use move semantics.